Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs

```diff
@@ -370,30 +370,48 @@ private void UnityBuildGUI()
 
                 using (new EditorGUILayout.HorizontalScope())
                 {
-                    // If the WSA target device is HoloLens, show the checkboxes for research mode
-                    if (EditorUserBuildSettings.wsaSubtarget == WSASubtarget.HoloLens)
+                    using (new EditorGUILayout.VerticalScope())
                     {
-                        // Enable Research Mode Capability
-                        bool curResearchModeCapabilityEnabled = UwpBuildDeployPreferences.ResearchModeCapabilityEnabled;
-                        bool newResearchModeCapabilityEnabled = EditorGUILayout.ToggleLeft(researchModeCapabilityLabel, curResearchModeCapabilityEnabled);
-
-                        if (newResearchModeCapabilityEnabled != curResearchModeCapabilityEnabled)
+#if !UNITY_2019_1_OR_NEWER
+                        // If the WSA target device is HoloLens, show the checkboxes for research mode
```

Comment:
should we consider hiding this checkbox if ar foundation packages are installed? they do not build correctly using the .net backend if c# projects are generated

Reply:
We could, though it wouldn't prevent people from checking it in the normal Unity build window instead (which would affect this build as well).

I can make sure the build failure message is clear and actionable about the failure, at the very least.

I'm also not fully sure how we'd detect those packages. I forgot, do you detect them elsewhere a specific way?

Reply:
> We could, though it wouldn't prevent people from checking it in the normal Unity build window instead (which would affect this build as well).

agreed

> I'm also not fully sure how we'd detect those packages. I forgot, do you detect them elsewhere a specific way?

I have code in the EditorProjectUtilities class that searches the Library\Packages folder.